userspace/libscap/scap.c

```diff
@@ -2492,7 +2497,9 @@ int64_t scap_get_readfile_offset(scap_t* handle)
 }
 
 #ifndef CYGWING_AGENT
-static int32_t scap_handle_eventmask(scap_t* handle, uint32_t op, uint32_t event_id)
+int32_t scap_handle_eventmask(scap_t* handle, uint32_t op, uint32_t event_id);
+
+int32_t scap_handle_eventmask__impl(scap_t* handle, uint32_t op, uint32_t event_id)
```

Comment:
Why do we have both `scap_handle_eventmask` and `scap_handle_eventmask__impl`?

Reply:
`scap_handle_eventmask` is a public-ish API which only passes through to the vtable (it's defined in scap_vtable.c), while `scap_handle_eventmask__impl` is the actual concrete implementation.

Whenever a function below calls `scap_handle_eventmask()`, it looks up the function pointer in the vtable and eventually ends up calling `scap_handle_eventmask__impl`, so this change is a no-op functionally.

Does this help?

Reply:
I see, thanks!